pkg/operator/starter.go

```diff
@@ -87,6 +88,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		workloadcontroller.EnsureAtMostOnePodPerNode,
 		kubeClient,
 		controllerConfig.EventRecorder,
+		configLister,
```

Comment:
`configInformers.Config().V1().ClusterVersions().Lister()` can be used directly

Reply:
done